### PR TITLE
monsters documentation has been updated

### PR DESCRIPTION
**Pull request by AnonymousICT**
## What does this do?
updates documentation to reflect what appears in the api

## How was it tested?
N/A

## Is there a Github issue this is resolving?
issue #57 

## Here's a fun image for your troubles
Max 025

### CODE REVIEW

File: src/views/partials/doc-resource-monsters.ejs [hunk 1]
```diff
@@ -191,5 +180,165 @@
...
+      <td align="left">string</td>
+    </tr>
+    <tr>
+      <td align="left">name</td>
+      <td align="left">The name of the monster</td>
+      <td align="left">string</td>
+    </tr>
+    <tr>
+      <td align="left">size</td>
+      <td align="left">The size of the monster ranging from tiny to Gargantuan</td>
```
Comment (bagelbits) [inline src/views/partials/doc-resource-monsters.ejs:196]: Nitpick: I'd probably capitalize `Tiny`.

File: src/views/partials/doc-resource-monsters.ejs [hunk 2]
```diff
@@ -191,5 +180,165 @@
...
+    <tr>
+      <td align="left">subtype</td>
+      <td align="left">The sub-category of a monster used for classification of monsters</td>
+      <td align="left">string</td>
+    </tr>
+    <tr>
+      <td align="left">alignment</td>
+      <td align="left">
+        The temperment of a monster. How likily will it go on a rampage versus a creature that is
+        able to negotiate for its own survival
```
Comment (bagelbits) [inline src/views/partials/doc-resource-monsters.ejs:213]: I might actually rewrite this as:
```
A creature's general moral and personal attitudes.
```

File: src/views/partials/doc-resource-monsters.ejs [hunk 3]
```diff
@@ -191,5 +180,165 @@
...
+
+    </tr>
+    <tr>
+      <td align="left">damage_vulnerabilities</td>
+      <td align="left">A list of damage types that a monster will take double damage from</td>
+      <td align="left">list</td>
+      
+    </tr>
+    <tr>
+      <td align="left">damage_resistances</td>
```
Comment (bagelbits) [inline src/views/partials/doc-resource-monsters.ejs:288]: Are we missing `damage_immunities`?

File: src/views/partials/doc-resource-subclasses.ejs [hunk 4]
```diff
@@ -15,7 +14,128 @@
   "desc": [
     "The archetypal Champion focuses on the development of raw physical power honed to deadly perfection. Those who model themselves on this archetype combine rigorous training with physical excellence to deal devastating blows."
   ],
-  "features": [],
+  "spells": [
```
Comment (bagelbits) [inline src/views/partials/doc-resource-subclasses.ejs:17]: Should we switch this to the Paladin subclass instead so it's consistent?

File: src/views/partials/doc-resource-monsters.ejs (hunk 1 above)
Comment (AnonymousICT) [reply src/views/partials/doc-resource-monsters.ejs:196]: will do

File: src/views/partials/doc-resource-classes.ejs [hunk 5]
```diff
@@ -368,68 +337,105 @@
...
 </table>
 
 <h3>GET api/classes/{index}/spellcasting/</h3>
 
 <pre><code>{
+  "index": "bard",
   "class": {
-    "url": "/api/classes/bard",
-    "name": "Bard"
+    "url": "url/classes/bard",
```
Comment (bagelbits) [inline src/views/partials/doc-resource-classes.ejs:386]: Should this change?

File: src/views/partials/doc-resource-monsters.ejs (hunk 2 above)
Comment (AnonymousICT) [reply src/views/partials/doc-resource-monsters.ejs:213]: done

File: src/views/partials/doc-resource-monsters.ejs (hunk 3 above)
Comment (AnonymousICT) [reply src/views/partials/doc-resource-monsters.ejs:288]: whoopsie doodle. let me add that in

File: src/views/partials/doc-resource-subclasses.ejs (hunk 4 above)
Comment (AnonymousICT) [reply src/views/partials/doc-resource-subclasses.ejs:17]: I have swapped the example to be consistent

File: src/views/partials/doc-resource-monsters.ejs (hunk 3 above)
Comment (AnonymousICT) [reply src/views/partials/doc-resource-monsters.ejs:288]: I have added damage_immunites to the table

File: src/views/partials/doc-resource-classes.ejs (hunk 5 above)
Comment (AnonymousICT) [reply src/views/partials/doc-resource-classes.ejs]: it does seem a bit redundant

Comment (bagelbits) [reply src/views/partials/doc-resource-classes.ejs]: Looks like it is `"url": "/api/classes/bard",` in the API?

File: src/views/partials/doc-resource-classes.ejs [hunk 6]
```diff
@@ -384,7 +384,6 @@
   "index": "bard",
   "class": {
     "url": "url/classes/bard",
-    "name": "Bard",
   },
```
Comment (bagelbits) [inline src/views/partials/doc-resource-classes.ejs:387]: Looks like this section should be:
```
  "class": {
    "url": "/api/classes/bard",
    "name": "Bard"
  },
```

Comment (AnonymousICT) [reply src/views/partials/doc-resource-classes.ejs]: got it

**Pull request merged**